src/components/common/FooterMenu.js

```diff
@@ -168,6 +176,15 @@ const FooterMenu = ({
                   {WORD_MODE_LABEL}
                 </span>
               </IconButton>
+              {isWordGameMode && (
+                <IconButton onClick={toggleUltraZenMode}>
+                  <Tooltip title={ULTRA_ZEN_MODE}>
+                    <span className={getModeButtonClassName(isUltraZenMode)}>
+                      <ZoomInMapIcon fontSize="medium" />
```

Comment:
```suggestion
                      <ZoomInMapIcon fontSize="small" />
```

your call lol, let me know what do you think